{{cookiecutter.project_slug}}/docs/source/index.rst

```diff
@@ -10,6 +10,7 @@
    dev_guide
    processes
    changes
+   authors
```

Comment:
Since authors file is optional, this probably should within an if-block.  I guess bug from upstream.